BLE-BCI-Car-Remote/README.md

```
@@ -0,0 +1,180 @@
+
+---
+
+## 5. BLE Control Protocol
+
+### BLE Service
+
+* Custom BLE service UUID
+* Two characteristics:
+
+| Characteristic | Purpose                 |
+|---------------|-------------------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)